### PR TITLE
Refactoring reliability and security

### PR DESCRIPTION
**Pull request by rfpcmaia**
Code improvements on reliability and security based on SonarQube suggestions.

### CODE REVIEW

File: app/css/Evaluation.css [hunk 1]
```diff
@@ -25,11 +25,9 @@ body {
 .btn {
   margin-top: 10px;
   padding: 10px 20px;
-  font-size: 16px;
```
Comment (Nasibgz) [inline app/css/Evaluation.css:28]: change the 16px with 1rem as it is the same by default and is better for scalability

File: app/css/MainPage.css [hunk 2]
```diff
@@ -83,11 +82,9 @@ main {
 .btn-upload {
   margin-top: 10px;
   padding: 10px 20px;
-  font-size: 16px;
```
Comment (Nasibgz) [inline app/css/MainPage.css:86]: same problem, I would change the 16px with 1rem like so:
font-size: 1rem;
for better scalability and understandability as it is the same by default

File: app/css/LoginStyle.css [hunk 3]
```diff
@@ -43,16 +39,12 @@ body {
 }
 
 .input-box input {
-
-    border: none;
```
Comment (Nasibgz) [inline app/css/LoginStyle.css:47]: Add a :focus style with a visible outline or other visual indicator for better accessibility accessibility

File: app/css/ViewEvaluation.css [hunk 4]
```diff
@@ -1,6 +1,6 @@
 body {
   font-family: Arial, sans-serif;
-  background-color: burlywood; /* Updated */
```
Comment (Nasibgz) [inline app/css/ViewEvaluation.css:3]: two lines of code for the background color when only one is needed. in other words, it is better we remove this line of code as it serves no purpose

**Review by ConUNony — COMMENTED** [review]
Security Improvement: Hiding version information from end users in app.js.
Code Refactoring: Replacing direct mysql2 mock with db config mock in app/test.js.
CSS Adjustments: Removing redundant styles and improving layout consistency.

Suggestions:

CSS Scalability: Change font-size from 16px to 1rem for better scalability.
Accessibility: Add a :focus style with a visible outline or other visual indicators for better accessibility.

overall looks good with no major issues

**Pull request merged**